docs/language/operators/summarize.md

```diff
@@ -5,26 +5,29 @@
 ### Synopsis
 
 ```
-summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>]] [by [<field>][:=<expr>] ...]
+summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...] [by [<field>][:=<expr>] ...]
 ```
 ### Description
 
-The `summarize` operator consumes all of its input, applies an aggregate function
+The `summarize` operator consumes all of its input, applies an [aggregate function](../aggregates/README.md)
 to each input value optionally organized with the group-by keys specified after
-the `by` keyword and at the end of input, and produces one or more aggregations
+the `by` keyword, and at the end of input produces one or more aggregations
 for each unique set of group-by key values.
 
-Each aggregate function may be optionally followed by a `where` clause, which
-indicates a Boolean expression that indicates, for each input value,
-whether to deliver it to that aggregate.
+The keyword `summarize` can be treated as an [implied operator](../overview.md#26-implied-operators)
+and hence may be omitted before an aggregate function call.
+
+Each aggregate function may be optionally followed by a [`where`](where.md) clause, which
+applies a Boolean expression that indicates, for each input value,
+whether to deliver it to that aggregate function.
 
 The output field names for each aggregate and each key are optional.  If omitted,
-a field name is inferred from each right-hand side, e.g, the output field for the `sum`
-aggregate function is simply `sum`.
+a field name is inferred from each right-hand side, e.g, the output field for the [`count`](../aggregates/count.md)
+aggregate function is simply `count`.
```

Comment:
I chose to reference `count` since it's one more opportunity to avoid any new user confusion about `summarize` vs. `sum`.